src/main/resources/application.yaml

```diff
@@ -143,7 +145,7 @@ old-letter-content-cleanup:
 
 encryption:
   enabled: ${ENCRYPTION_ENABLED:false}
-  publicKey: ${ENCRYPTION_PUBLIC_KEY:}
+  publicKey: ${ENCRYPTION_PUBLIC_KEY}
```

Comment:
will this cause problems with local dev?

Reply:
I have reverted, this is needed for local run.